src/concepts/external_data_integration.md

```diff
@@ -0,0 +1,75 @@
+
+Fivetran can be accessed using LDAP: `fivetran.com/dashboard`
+
+## Architecture
+
+Fivetran is used to extract data from external services and load it into our data warehouse. For transforming data or writing data to a different destination [bigquery-etl](https://github.com/mozilla/bigquery-etl) is used.
+
+The integration of Fivetran and external data import into our data platform is illustrated in the following:
+
+![](../assets/fivetran_architecture.png)
+
+A separate connector is set up in Fivetran for each external service data should be imported from. The imports get triggered via Airflow which will have different DAGs with a specific schedule. Fivetran will dump the data into our data warehouse. Once the first airflow-triggered import is done, Airflow can trigger the ETL process which will write the transformed data to a different location in our data warehouse.
+
+### Environments
+
+We have a production and development environment in Fivetran.
+
+| Environment | Description                                                                                                                               | Fivetran Warehouse | Data Destination          | Access                                                                                             |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------ | ------------------------- | -------------------------------------------------------------------------------------------------- |
+| Development | This environment allows developers to test their connectors during development, quickly deploy changes and make changes to imported data. | **Fivetran_Dev**   | `dev-fivetran`            | Anyone in data engineering                                                                         |
+| Production  | This environment has connectors deployed that are used in production. Access to connectors and imported data is restricted.               | **Fivetran_Prod**  | `moz-fx-data-bq-fivetran` | Only automation has access. If access to the data is required a ticket needs to be filed with SRE. |
+
+To prevent Fivetran from having access to our data warehouse, separate data destinations have been provisioned for Fivetran to dump imported data into. The data can be transformed and moved to different data destinations using bigquery-etl.
+
+When working in the **Fivetran_Dev** environment, data is written to `dev-fivetran`. All datasets are by default accessible to data engineers. So when working with sensitive data, it might be necessary to manually update the access permissions to the dataset. Ensure that the Fivetran service account has the necessary permissions to write data to the dataset when making updates.
```

Comment:
Can we be explicit here that we mean GCP project `dev-fivetran` in BigQuery?
```suggestion
When working in the **Fivetran_Dev** environment, data is written to `dev-fivetran` GCP project in BigQuery. All datasets are by default accessible to data engineers. So when working with sensitive data, it might be necessary to manually update the access permissions to the dataset. Ensure that the Fivetran service account has the necessary permissions to write data to the dataset when making updates.
```